SessionSnodeKit/Types/Network.swift

```diff
@@ -61,12 +61,20 @@ public extension Network {
         fileprivate static let legacyFileServer = "http://88.99.175.227"
         fileprivate static let legacyFileServerPublicKey = "7cb31905b55cd5580c686911debf672577b3fb0bff81df4ce2d5c4cb3a7aaa69"
         
+        static let networkAPIServer = "http://networkv1.getsession.org"
+        static let networkAPIServerPublicKey = "cbf461a4431dc9174dceef4421680d743a2a0e1a3131fc794240bcb0bc3dd449"
```

Comment:
Since this is using a different domain it might be better to move it into it's own enum (it's possible in the future that it wouldn't be hosted on the `FileServer` which makes this a little misleading)